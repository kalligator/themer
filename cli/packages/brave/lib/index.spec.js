const { render, renderInstructions } = require('./index');
const { colors } = require('../../colors-default');

describe('Brave theme generator', () => {
  it('should provide installation instructions', async () => {
    const files = await Promise.all(render(colors));
    const instructions = renderInstructions(files.map(({ name }) => name));
    expect(instructions).toMatchSnapshot();
  });
});
