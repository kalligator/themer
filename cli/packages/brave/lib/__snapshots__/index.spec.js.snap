// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brave theme generator should provide installation instructions 1`] = `
"
1. Launch Brave and go to \`brave://extensions\`.
2. Check the \\"Developer mode\\" checkbox at the top.
3. Click the \\"Load unpacked extension...\\" button and choose the desired theme directory (\`Themer Dark\` or \`Themer Light\`).

(To reset or remove the theme, visit \`brave://settings\` and click \\"Reset to Default\\" in the \\"Appearance\\" section.)
  "
`;
